webpack.config.js

```diff
@@ -0,0 +1,23 @@
+var Encore = require('@symfony/webpack-encore');
+var path = require('path');
+
+Encore
+    .setOutputPath('src/Resources/public/dist')
+    .setPublicPath('/bundles/sonataadmin/dist')
+    .setManifestKeyPrefix('dist')
+    .addEntry('sonata_admin', './src/Resources/public/js/sonata_admin.js')
+    .autoProvidejQuery()
+    .enableSassLoader()
+    .enableSingleRuntimeChunk()
+    .splitEntryChunks()
+    .enableSourceMaps(!Encore.isProduction())
+    .cleanupOutputBeforeBuild()
+    .enableBuildNotifications()
+    .addAliases({
+        jquery: path.resolve(__dirname, 'node_modules/jquery/dist/jquery')
+    })
+    //.splitEntryChunks()
```

Comment:
Maybe you can add a copy job, so you can copy all locale files to the dist dir and create an async loading for this?

```js
  .copyFiles({
    from: './node_modules/select2/',
    pattern: /select2_locale_.*\.js$/,
    to: 'select2/locale/[name].[ext]'
  })
```

Reply:
I agree with that but async is not the problem. It will work for `select2_locale_de.js`, `select2_locale_en.js` but whats with `select2_locale_zh-CN.js`?

Reply:
That's a problem of the plugin. Do they have a solution?

Reply:
i think this is our problem. because of this PR https://github.com/sonata-project/SonataAdminBundle/pull/4878 

i mean currently the localizing feature works. The exception are `zh-CN.js`, `zh-TW.js` and so on. the rest  of the list works and no they don't have a solution:

take a look at the list https://github.com/select2/select2/tree/develop/src/js/select2/i18n

i think there is no reason for a mapping almost every language is loaded and supported execpt a few.

Reply:
In my opinion it should work like expected. After this branch is merged i would like to update  some frontend packages (use maybe vuejs for select2, calendar picker etc.) Dream would be to remove jQuery

Reply:
Maybe this will help?
https://github.com/jmblog/how-to-optimize-momentjs-with-webpack